### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by JayRajasekharan**
Added following links: 
How to Write a Spelling Corrector - https://norvig.com/spell-correct.html
DataCamp: Advanced NLP with spaCy - https://www.datacamp.com/courses/advanced-nlp-with-spacy
Pattern.en - https://www.clips.uantwerpen.be/pages/pattern-en
BERT Explained  -https://towardsdatascience.com/bert-explained-state-of-the-art-language-model-for-nlp-f8b21a9b6270

Fixed the following:
Replaced link for "Understanding Convolutional Neural Networks for NLP" with cached version as the wildml.com domain is not longer active.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ A curated list of resources for learning about natural language processing, text
 - [Machine Learning for Text (2018)](https://www.springer.com/us/book/9783319735306)
 - [Towards Explainable NLP: A Generative Explanation Framework for Text Classification](https://arxiv.org/pdf/1811.00196.pdf)
 - [Natural Language Processing in Spanish](https://iaarbook.github.io/procesamiento-del-lenguaje-natural/)
+- [How to Write a Spelling Corrector - by Peter Norvig](https://norvig.com/spell-correct.html)
```
Comment (stepthom) [inline README.md]: Very interesting post, but I'm not sure it's a book. May be better under Blog Articles.

File: README.md [hunk 2]
```diff
@@ -294,9 +295,10 @@ Errors in Data](https://journal.r-project.org/archive/2010-2/RJournal_2010-2_Sar
 - [Contextual String Embeddings for Sequence Labeling](http://alanakbik.github.io/papers/coling2018.pdf). 2018.
 - [A Hierarchical Multi-task Approach for Learning Embeddings from Semantic Tasks](https://arxiv.org/abs/1811.06031) Introducing a multi-task learning approach for a set of interrelated NLP tasks. Presented at AAAI conference in January 2019.[Implementation code](https://github.com/huggingface/hmtl).  
 - [An Idiot’s Guide to Word2vec Natural Language Processing](https://medium.com/@ODSC/an-idiots-guide-to-word2vec-natural-language-processing-5c3767cf8295)
+- [BERT Explained - State of the art language model for NLP](https://towardsdatascience.com/bert-explained-state-of-the-art-language-model-for-nlp-f8b21a9b6270). A great explaination of the fundamentals of how BERT works.
 
 ### Deep Learning
-- [Understanding Convolutional Neural Networks for NLP](http://www.wildml.com/2015/11/understanding-convolutional-neural-networks-for-nlp/). 
+- [Understanding Convolutional Neural Networks for NLP](http://webcache.googleusercontent.com/search?q=cache:http://www.wildml.com/2015/11/understanding-convolutional-neural-networks-for-nlp/).
```
Comment (stepthom) [inline README.md]: The webcache link doesn't work for me. Since wildml is down, maybe just remove this link altogether.

File: README.md [hunk 3]
```diff
@@ -399,6 +402,7 @@ Errors in Data](https://journal.r-project.org/archive/2010-2/RJournal_2010-2_Sar
   - [textblob](https://textblob.readthedocs.io/en/dev/): Simplified Text processing.
     - [Natural Language Basics with TextBlob](http://rwet.decontextualize.com/book/textblob/)
   - [Gensim](https://radimrehurek.com/gensim/): Topic Modeling for humans.
+  - [Pattern.en](https://www.clips.uantwerpen.be/pages/pattern-en): The pattern.en module contains a fast part-of-speech tagger for English, sentiment analysis, tools for English verb conjugation and noun singularization & pluralization, and a WordNet interface.
```
Comment (stepthom) [inline README.md]: This is great. For consistency, please remove the text "The pattern.en module contains"

File: README.md (hunk 1 above)
Comment (JayRajasekharan) [reply README.md]: Moved to Blog Articles, Papers, Case Studies

File: README.md (hunk 2 above)
Comment (JayRajasekharan) [reply README.md]: Removed the the link

File: README.md (hunk 3 above)
Comment (JayRajasekharan) [reply README.md]: Removed the text

**Pull request merged**